DemoApp/DemoApp/Views/TabViewItems/TabViewFour.xaml.cs

```diff
@@ -0,0 +1,13 @@
+﻿using DemoApp.ViewModels;
+
+namespace DemoApp.Views.TabViewItems;
+
+public partial class TabViewFour : ContentView
+{
+    public TabViewFour()
+    {
+        InitializeComponent();
+        BindingContext = new SegmentedControlViewModel();
```

Comment:
BindingContext is not setting right. As a result, changing anything in the view doesn't change the appearance of the controls.
<img width="1309" alt="Screen Shot 2023-08-08 at 10 11 38 PM" src="https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/c4bb6a7f-864b-48ec-af86-61959dd1aff4">

From the logs:

> Microsoft.Maui.Controls.Xaml.Diagnostics.BindingDiagnostics: Warning: 'CurrentTextColor' property not found on 'Trimble.Modus.Components.TabViewItem', target property: 'Microsoft.Maui.Controls.Label.TextColor'
Microsoft.Maui.Controls.Xaml.Diagnostics.BindingDiagnostics: Warning: 'RadioButtons' property not found on 'DemoApp.ViewModels.SpinnerPageViewModel', target property: 'Trimble.Modus.Components.TMRadioButtonGroup.ItemsSource'
Microsoft.Maui.Controls.Xaml.Diagnostics.BindingDiagnostics: Warning: 'CurrentIcon' property not found on 'Trimble.Modus.Components.Model.SegmentItemModel', target property: 'Microsoft.Maui.Controls.Image.IsVisible'

Reply:
Set Binding Context as values are working now

Reply:
Tested after adding commands the context is accessible it works.